### ISSUE TITLE
Implement the new Export Tool User Interface (UI) design

### ISSUE THREAD

**Issue by mataharimhairi** [hotosm/hotosm-project-ideas#16]
**Proposed by:** Mhairi O'Hara     

**Summary:** The HOT Export Tool UI needs to be redesigned to increase overall usability and efficiency. 

**Skills Required:** jQuery, OpenLayers, HTML5, CSS, UI design

**Possible Mentors:** Dražen Odobašić

**Notes:** The [HOT Exports Tool](http://export.hotosm.org/en) will be enhanced by implementing th new UI design, making it easier and quicker to use, so that data can be exported more efficiently. This is listed in the [hot-exports](https://github.com/hotosm/hot-exports) GitHub repository as [Issue #68](https://github.com/hotosm/hot-exports/issues/68) Proposing UI changes.

**Comment by mataharimhairi** [comment]
@cgiovando given that the internship is only for 3 months, I will update this project idea to solely address [issue #68](https://github.com/hotosm/hot-exports/issues/68) Proposing UI changes, removing [issue #14](https://github.com/hotosm/hot-exports/issues/14) completely and leave [issue #78](https://github.com/hotosm/hot-exports/issues/78) listed as a project idea in this repository as issue #17.

**Comment by wonderchook** [comment]
I think this should be more "implement UI" changes instead of redesign. There has been a redesign, but someone needs to turn that into code.

**Comment by cgiovando** [comment]
@mataharimhairi can you find one or two mentors?

**Comment by mataharimhairi** [comment]
@dodobas would you be willing to mentor this project as well? I've but you down for #17. I'll mentor any Export Tool project selected by the Outreachy program and was wondering if you could cover the Export Tool projects for GSoC? There are currently 2.

**Comment by dodobas** [comment]
well, I'm not really an UI guy, so can't really help with the design or usability, but I can help with all of the other stuff, and if that's ok, then yes

**Comment by bgirardot** [comment]
@mataharimhairi Are any of these export tool projects now subsumed by the hot export tech challenge?

**Comment by mataharimhairi** [comment]
@bgirardot Yes the export tool projects that were originally going to be proposed in GSoC are subsumed by the HOT Ecport Tech Challenge:

http://hot.openstreetmap.org/hot_export_tool_tech_challenge
